scripts/check_phase44c_projection_root_probe.py

```
@@ -0,0 +1,306 @@
+        "probe": PROBE,
+        "source_surface_version": manifest["source_surface_version"],
+        "source_mechanics": source_mechanics,
+        "projection_row_count": result["projection_row_count"],
+        "projection_log_size": result["projection_log_size"],
+        "source_emitted_projection_root": source_emitted_projection_root,
+        "canonical_source_root": result["canonical_source_root"],
+        "kill_results": kill_results,
+    }
+
+
+def write_json(path: pathlib.Path, value: Any) -> None:
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)